application/src/main/java/org/togetherjava/tjbot/features/moderation/TransferQuestionCommand.java

```diff
@@ -48,6 +48,7 @@ public final class TransferQuestionCommand extends BotCommandAdapter
     private static final int TITLE_COMPACT_LENGTH_MAX = 30;
     private final Predicate<String> isHelpForumName;
     private final List<String> defaultTags;
+    private static final Color TURQUOISE = new Color(50, 164, 168);
```

Comment:
bad name. instead of saying what color it is (we can all see that in intellij), say what its used for. in this case, it is the color for ur embed. so `EMBED_COLOR` or similar would be better, thanks